util/repeatable_thread.h

```diff
@@ -15,6 +15,11 @@
 
 namespace rocksdb {
 
+/**
+ * Simple wrapper around port::Thread that supports calling a callback every
+ * X seconds. If you pass in 0, then it will call your callback repeatedly
+ * without delay.
+ */
```

Comment:
Can we keep the format consistent to other part of the code? I believe we use "//" everywhere.